kettle/make_db.py

```diff
@@ -66,7 +66,8 @@ def _request(self, path, params, as_json=True):
                     try:
                         return resp.json()
                     except json.decoder.JSONDecodeError:
-                        logging.exception('Failed to decode request for %s', path)
+                        logging.error('Failed to decode request for %s',
+                                      urllib.parse.unquote(url))
```

Comment:
AFAICT this is the only real change in this PR, it's unclear to me how it's related to the description?

Reply:
Previously it was logging the stack with the log. It made me believe that there was a stack trace being hit because there was no real indication that it was part of the log message. The error is common to all jobs where they have a blank started/finsihed.json. So spamming the logs with traces was obnoxious and I changes it. When I say "trace back logging" i just mean that the trace was being logged to console.

Reply:
🤦 I missed the change from `error` to `exception` and was only looking at the parameters.  This makes sense